pyproject.toml

```diff
@@ -0,0 +1,78 @@
+[project]
+dynamic = ["version", "dependencies"]
+name = "kinto-emailer"
+description = "Kinto emailer plugin"
+readme = {file = "README.rst", content-type = "text/x-rst"}
+license = {file = "LICENSE"}
+classifiers = [
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Topic :: Internet :: WWW/HTTP",
+    "Topic :: Internet :: WWW/HTTP :: WSGI :: Application",
+    "License :: OSI Approved :: Apache Software License"
+]
+keywords = ["web services"]
+authors = [
+  {name = "Mozilla Services", email = "developers@kinto-storage.org"},
+]
+
+[tool.setuptools.packages.find]
+include = ["kinto_emailer"]
+
+[project.urls]
+Repository = "https://github.com/Kinto/kinto-emailer"
+
+[project.scripts]
+kinto-send-email = "kinto_emailer.command_send:main"
+
+[build-system]
+requires = ["setuptools>=64", "setuptools_scm>=8"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools.dynamic]
```

Comment:
Nit, but could we move some of these tables around so that `project.*` are closer to the project table, and `tool.setuptools.*` are closer to `build-system`?